addon/components/ember-tooltip-base.js

```diff
@@ -238,14 +240,6 @@ export default Component.extend({
     this._dispatchAction('onDestroy', this);
   },
 
-  addTargetEventListeners() {
```

Comment:
I'm thinking we should keep these and `hide()` defined in the base component, as people may be extending `EmberTooltipBase` and depending on those implementations.

Reply:
Fair enough.